packages/antlr4.rb

```diff
@@ -51,8 +51,7 @@ def self.install
   end
 
   def self.postinstall
-    puts "\nTo finish the installation, execute the following:".lightblue
-    puts 'source ~/.bashrc'.lightblue
-    puts "\nType 'antlr4' to get started.\n".lightblue
+    ExitMessage.add "\nTo finish the installation, please execute the following:\nsource ~/.bashrc".lightblue, print_last: true
```

Comment:
Use `<<~EOT`?

Reply:
This is a one liner, so simpler?

Reply:
But ideally this can be extracted out into a macro...

Reply:
> This is a one liner, so simpler?

I always prefer using heredoc in messages as it makes source code more clear ("what you see is what you get") :)

> But ideally this can be extracted out into a macro...

Maybe we can add it to `package.rb`?

Reply:
I have ZERO objections to adding it to `package.rb`!

Reply:
Do you want to add it to this PR?

Reply:
> Do you want to add it to this PR?

Go ahead

Reply:
Maybe we can make it a property something like `print_source_bashrc_message`?

Reply:
working on adding it...

Reply:
@supechicken Remind  me how I make this have .lightblue again...
```
  if @pkg.reload_source?
    ExitMessage.add <<~EOT
      To finish the installation, please execute the following:
      source ~/.bashrc
    EOT
  end
```

Reply:
Submethods can be chained after `<<~EOT`:
```ruby
puts <<~EOT.lightblue
```